app.js

```diff
@@ -0,0 +1,17 @@
+const express = require("express");
+const app = express();
+app.use(express.json());
```

Comment:
Why do you have app.use(express.json()) at this point?

Please don't add things you don't need until you actually need them.